content/tutorials/r_paired_t_test/index.Rmd

```diff
@@ -0,0 +1,154 @@
+---
+title: "Mixed model formulation of a paired t-test"
+description: "Learn some insights about mixed models based on a simple example"
+author: "Hans Van Calster"
+date: "`r Sys.Date()`"
+categories: ["r", "statistics"]
+tags: ["r", "analysis", "mixed model"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+## Context
+
+The standard paired t-test is typically used to test for a significant differences between two paired treatments.
+We can formulate the test in terms of a mixed model.
+The benefit is that we get more informative model output, which allows us among other things to check if model assumptions were met.
+For the paired t-test, one assumption is that the paired differences between treatments follow a normal distribution.
```

Comment:
Maybe move this part to later in the document, after you've shown the equivalence between the paired t-test and the mixed model